test/SILOptimizer/definite_init_closures.swift

```diff
@@ -0,0 +1,50 @@
+// RUN: %target-swift-frontend -emit-sil %s -o /dev/null
+
+// Test boolean operators with implicit closures
```

Comment:
I need a sil test case before I can approve this.

Reply:
I added one.